<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface_Employe_RH</class>
 <widget class="QMainWindow" name="Interface_Employe_RH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des employés</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rssources.qrc">
    <normaloff>:/images/logo_erp.png</normaloff>:/images/logo_erp.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>460</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>400</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Traiter les données</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>611</width>
        <height>281</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Informations de travailleur</string>
      </property>
      <widget class="QTextEdit" name="textEdit_adresse">
       <property name="geometry">
        <rect>
         <x>335</x>
         <y>210</y>
         <width>181</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_tele">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_age">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>130</y>
         <width>42</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_CIN">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>12</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_CIN">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_etat_2">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>190</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Adresse</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_diplome">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Diplôme</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_email">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>40</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_poste">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>40</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Poste</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_email">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>40</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Email de travail</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_grade">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>40</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Grade</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_depard">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>100</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Démission</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Licenciement</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Retraite</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fin de contrat</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_cause_depard">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>100</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Cause de départ</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_departement">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>130</y>
         <width>75</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Département</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_etat">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>160</y>
         <width>96</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Active</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Inactive</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_etat">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>158</y>
         <width>28</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Etat</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prenom">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_prenom">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>70</y>
         <width>46</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Prénom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_nom">
       <property name="geometry">
        <rect>
         <x>31</x>
         <y>40</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_telephone">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>70</y>
         <width>62</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Téléphone</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_diplome">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Bac +3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bac +5</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_poste">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>220</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Directeur RH</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Directeur Financier</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Directeur Comptable</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Directeur Production</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Agent de production</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Agent de stock</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_grade">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>190</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Ouvrier</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cadre</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Stagiaire</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_departement">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>130</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>RH</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FI</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_emp_mo">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>370</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_emp_verifier">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_emp_aj">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>370</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_travailleur_id">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Travailleur Id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_age">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_emp_sup">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>370</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QFrame" name="logo_edf">
      <property name="geometry">
       <rect>
        <x>446</x>
        <y>0</y>
        <width>169</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/EdfErp.png);
image: url(:/images/logo_edf_erp.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>400</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Afficher les données</string>
     </attribute>
     <widget class="QTableView" name="tableView_employe">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>621</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_emp_aff">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Actualiser</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>lineEdit_nom</tabstop>
  <tabstop>lineEdit_prenom</tabstop>
  <tabstop>lineEdit_CIN</tabstop>
  <tabstop>spinBox_age</tabstop>
  <tabstop>comboBox_diplome</tabstop>
  <tabstop>comboBox_poste</tabstop>
  <tabstop>comboBox_grade</tabstop>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>lineEdit_tele</tabstop>
  <tabstop>comboBox_depard</tabstop>
  <tabstop>comboBox_departement</tabstop>
  <tabstop>comboBox_etat</tabstop>
  <tabstop>textEdit_adresse</tabstop>
  <tabstop>pushButton_emp_aj</tabstop>
  <tabstop>pushButton_emp_mo</tabstop>
  <tabstop>pushButton_emp_sup</tabstop>
  <tabstop>pushButton_emp_verifier</tabstop>
  <tabstop>tableView_employe</tabstop>
  <tabstop>pushButton_emp_aff</tabstop>
 </tabstops>
 <resources>
  <include location="rssources.qrc"/>
 </resources>
 <connections/>
</ui>
